infra-gen2/backends/api/apiMultiAuth/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "api-multi-auth",
   "version": "1.0.0",
-  "main": "index.js"
+  "main": "index.js",
+  "dependencies": {
+    "infra-common": "1.0.0"
```

Comment:
Q: I thought dependencies were not added to the individual package.json files?

Reply:
Good catch, this can be hoisted to be dep for all sub packages.